### PR TITLE
About#751

### LINKED ISSUE #751: Clicking about option intents to activity, should attach fragment
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore
    
### Feature Request

**Feature Description**

Use fragment when the 'About' option clicked instead of activity, just like is done for other event properties.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mishuvs**
Fixes #751 

Changes:
1. Deletes AboutEventActivity and its layout
2. Removes AboutEventActivity from `ActivityInstanceTest.java` and manifest
3. Changes MainActivity to attach AboutEventFragment

Screenshots for the change: 
aboutnew

### CODE REVIEW

**Comment by iamareebjamal** [comment]
just get activity in onDetach and reset the action bar

**Comment by mishuvs** [comment]
> just get activity in onDetach and reset the action bar

That'd fix only the title issue. I have used an Interface to add the drawer toggle and am hiding the main ActionBar when fragment gets attached.

Please review.

File: app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java [hunk 1]
```diff
@@ -39,7 +41,8 @@
 
 import dagger.Lazy;
 
-public class MainActivity extends BaseActivity<IMainPresenter> implements NavigationView.OnNavigationItemSelectedListener, IMainView {
+public class MainActivity extends BaseActivity<IMainPresenter> implements
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java]: This is a recipe to create God Object. About Fragment should be responsible for handling the action bar changes

**Comment by iamareebjamal** [comment]
@mishuvs Please update the PR

**Comment by mishuvs** [comment]
@iamareebjamal Please review.

**Pull request merged**